rust/template/differential_datalog/src/program/worker.rs

```diff
@@ -592,90 +627,83 @@ impl<'a> DDlogWorker<'a> {
     }
 }
 
-fn render_relation<'a>(
+// TODO: Add back regions for relations
```

Comment:
So we're basically back to where we were before this PR?

Reply:
No, it's PR still introduces a bunch of named operators

Reply:
Got it. The PR's been around for a while and I'm starting to forget what's in it :)